packages/viewport/src/test/with-viewport-match.js

```diff
@@ -1,68 +1,64 @@
 /**
  * External dependencies
  */
-import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
 
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
-import { useViewportMatch as useViewportMatchMock } from '@wordpress/compose';
+import { useViewportMatch } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import '../store';
 import withViewportMatch from '../with-viewport-match';
 
+jest.mock( '@wordpress/compose/src/hooks/use-viewport-match' );
```

Comment:
Alternatively, maybe we could be able to avoid mocking `useViewportMatch` and rather render a `useViewportMatch.__experimentalWidthProvider` provider in the tree.

Reply:
Considering that `withViewportMatch` depends on values returned by `useViewportMatch`, mocking the hook here makes sense to me.

Reply:
Yeah, I'm happy with the mock as one of the paths forward, just wanted to mention that there could be an alternative 👍 Feel free to ship as-is (I approved it already)